### ISSUE TITLE
SelectOneMenu: ItemLabel "" causes min heigth 

### ISSUE THREAD

**Issue by cadubad** [primefaces/primefaces#7076]
**Describe the defect**
When I put itemLabel empty, it causes the height of the component to be too small

**Environment:**
PF Version: 10.0.0-RC2
JSF + version: jsf-impl-2.3.9.SP04
Affected browsers: Chrome

**To Reproduce**

1 - insert an empty itemLabel on any p:selectOneMenu
2 - When executed, even with other items in the list, the height is almost zero

```
<p:selectOneMenu value="#{usuarioBean.usuario.pais}" 
converter="paisConverter">
   <f:selectItem itemLabel="" />
   <f:selectItems value="#{utilBean.listarPaises()}" />
   <p:ajax event="change" update="@form" ></p:ajax>
</p:selectOneMenu>
```

**Expected behavior**
The height was expected to be the same for all items.

**Possible solution/workaround**
change empty value to any value
```    <f:selectItem itemLabel="..." /> ```

**Comment by melloware** [comment]
Duplicate of https://github.com/primefaces/primefaces/issues/7044 fixed for 10.0.0 Final.

**Comment by cadubad** [comment]
Thanks!
